Comp2022/src/main/cpp/commands/AutoShoot.cpp

```diff
@@ -38,7 +38,7 @@ AutoShoot::AutoShoot(
     AddCommands(
         IntakeDeploy(true),
         AutoWait(drivetrain),
-        ShooterRun(Shooter::SHOOTERSPEED_FORWARD, shooter),
+        ShooterRun(Shooter::SHOOTERSPEED_LOWHUB, shooter),

```

Comment:
Is the purpose of making the setting shooting at low hub at the moment just for testing? Just wondering